plugins/callback/logstash.py

```diff
@@ -63,76 +102,96 @@
 
 
 class CallbackModule(CallbackBase):
-    """
-    ansible logstash callback plugin
-    ansible.cfg:
-        callback_plugins   = <path_to_callback_plugins_folder>
-        callback_whitelist = logstash
-    and put the plugin in <path_to_callback_plugins_folder>
-
-    logstash config:
-        input {
-            tcp {
-                port => 5000
-                codec => json
-            }
-        }
-
-    Requires:
-        python-logstash
-
-    This plugin makes use of the following environment variables or ini config:
-        LOGSTASH_SERVER   (optional): defaults to localhost
-        LOGSTASH_PORT     (optional): defaults to 5000
-        LOGSTASH_TYPE     (optional): defaults to ansible
-    """
 
     CALLBACK_VERSION = 2.0
     CALLBACK_TYPE = 'aggregate'
     CALLBACK_NAME = 'community.general.logstash'
     CALLBACK_NEEDS_WHITELIST = True
 
-    def __init__(self, display=None):
-        super(CallbackModule, self).__init__(display=display)
+    def __init__(self):
+        super(CallbackModule, self).__init__()
 
         if not HAS_LOGSTASH:
             self.disabled = True
-            self._display.warning("The required python-logstash is not installed. "
-                                  "pip install python-logstash")
+            self._display.warning("The required python3-logstash is not installed. "
+                                  "pip install python3-logstash")
```

Comment:
I guess you should mention that `python-logstash` should be used for Python 2, and `python3-logstash` for Python 3?